docs/src/examples/components/Dropdown/BestPractices/DropdownBestPractices.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react'
+
+import ComponentBestPractices from 'docs/src/components/ComponentBestPractices'
+
+const doList = [
+  {
+    key: 'one',
+    content:
+      'Provide getA11ySelectionMessage, getA11yStatusMessage, noResultsMessage and loadingMessage props to announce state changes correctly.',
```

Comment:
for the sake of maintaining consistency, lets enclose prop names (e.g. `getA11ySelectionMessage`), prop values (`warning`, `danger`)  as well as attribute names (e.g. `aria-label`) in back ticks (`). This will provide proper formatting for these tokens and improve readability for a client

Reply:
nit: 

> to announce state changes correctly

May suggest to rephrase a bit: 

> to visualize dropdown state correctly